File: Dart/Slice/shuffle_slice.dart
import "dart:core";

void main() {
  // Create a new list of strings.
  final List<String> randomSlice = <String>["Bob", "Alice", "Joe"];
  // Shuffle the slice.
  print(shuffleASlice(randomSlice));
}

// Shuffle a given slice and return the result.
List<dynamic> shuffleASlice(final List<dynamic> slice) {
  slice.shuffle();
  return slice;
}

File: Python/Slice/shuffle_slice.py
import random

# Shuffle an slice.
def shuffle_slice(provided_slice):
    random.shuffle(provided_slice)
    return provided_slice

def main():
    # Make a slice using a random set of data.
    randomSlice = ["c", "a", "b"]
    # Shuffle an slice.
    print(shuffle_slice(randomSlice))

main()
